tests/tests/test-crowdloan.ts

```diff
@@ -48,23 +49,28 @@ describeDevMoonbeam("Crowdloan", (context) => {
       GENESIS_ACCOUNT
     );
     expect(isPayable.toHuman()).to.equal(null);
-    // check vesting period
-    expect((await context.polkadotApi.consts.crowdloanRewards.vestingPeriod).toHuman()).to.eq(
-      "201,600"
-    );
   });
   it("should be able to register the genesis account for reward", async function () {
     // should be able to register the genesis account for reward
     await context.polkadotApi.tx.sudo
       .sudo(
-        context.polkadotApi.tx.crowdloanRewards.initializeRewardVec(
-          [[relayChainAddress, GENESIS_ACCOUNT, 3_000_000n * GLMR]],
-          0,
-          1
-        )
+        context.polkadotApi.tx.crowdloanRewards.initializeRewardVec([
+          [relayChainAddress, GENESIS_ACCOUNT, 3_000_000n * GLMR],
+        ])
       )
       .signAndSend(sudoAccount);
     await context.createBlock();
+
+    let initBlock = (await context.polkadotApi.query.crowdloanRewards.initRelayBlock()) as any;
+
+    // Complete initialization
+    await context.polkadotApi.tx.sudo
+      .sudo(
+        context.polkadotApi.tx.crowdloanRewards.completeInitialization(Number(initBlock) + vesting)
```

Comment:
so 2 tx are needed to initialize a crowdloan?

Reply:
Yes, we changed the initialization to a 2 step extrinsic